compat/compat.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package compat.pb;
```

Comment:
Nice! This is a great technique we should keep using across the board on protobuf changes ;-)